components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -597,8 +604,9 @@ interface PlacesManager {
      * It has been designed exclusively for non-sync users.
      *
      * @param path Path to the `browser.db` file database.
+     * @return A list of pinned websites.
```

Comment:
Would you mind expanding the doc comment above to explain how Android stores top sites, and that we don't import them because we don't support creating new roots? It would help clarify why we return them to the caller instead of importing them like the others. Let's also clarify that they aren't imported into the DB at all, so callers can't use the GUID to, say, update or delete them.